src/components/ItemCard.js

```diff
@@ -1,13 +1,34 @@
 import "./ItemCard.css";
+import ChCakeImg from "../images/chocolate.png";
+import { RiFilePaper2Line } from "react-icons/ri";
+import { Link } from "react-router-dom";
 
-export default function ItemCard(props) {
+export default function ItemCard(props, handleDelete) {
+  function handleDelete() {
+    console.log("Clicked");
+  }
   return (
     <div className="ItemCard box">
-      <li className="ItemCard">
-        {/* <img src={props.image} alt="description" className="ItemCard__image" /> */}
+      <li className="ItemCard" key={props.id}>
+        <img src={ChCakeImg} alt="description" className="ItemCard__image" />
 
         <h2 className="ItemCard__name">{props.name}</h2>
-        <p className="ItemCard__ingredients">{props.ingredients}</p>
+        <div className="ItemCard--btnDiv">
+          <button
+            className="ItemCard__button delete"
+            onClick={() => handleDelete(props.id)}
```

Comment:
you are using an inline function here but also declaring the function above. I think in general the cleanest way is to declare the function in the logic part of the component and just call it with `onClick={handleDelete}`.

Reply:
Agree. I left if as I was wrapping the whole item with a Link and was testing if it would work.